recipe/meta.yaml

```diff
@@ -88,8 +88,8 @@ source:
     - two-patches.patch
 
 build:
-  number: 2
-  skip: true # [py<36]
+  number: 3
+  skip: true # [py<37]
```

Comment:
why `py<37`? Is this coming from upstream?

Reply:
cannot find anything about python requirements, but probably we don't want to support so ancient version and/or spoil snowflake channel